### ISSUE TITLE
ENH: Add sinc

### ISSUE THREAD

**Issue by jakirkham** [dask/dask#2311]
Would be nice to include the [`sinc`]( https://docs.scipy.org/doc/numpy-1.12.0/reference/generated/numpy.sinc.html#numpy.sinc ) function from `numpy`.

**Comment by mrocklin** [comment]
As with any of these ufuncs, this sounds good to me.

**Comment by mrocklin** [comment]
Are there others that we're missing?

**Comment by jakirkham** [comment]
Cool.

Yeah, there are a few.

For instance `true_divide` and `floor_divide`. Those in particular are handy if one wants to ensure a particular kind of division in Python 2/3 code.

Also `negative`, `power`, `remainder`, `mod`, but I don't know that any of these is more convenient than just using an operator. Still maybe handy for compatibility with NumPy. Considering `absolute` already exists in Dask Array, it seems like we are ok adding these.

Then some other miscellaneous functions like `exp2`, `cbrt`, and `reciprocal`, which are sometimes used, but not very often. Also there are other ways to get at these as well.

**Comment by mrocklin** [comment]
Presumably these are trivial to implement.  Probably just adding the
appropriate line to ufuncs.py and a corresponding line to tests?

**Comment by jakirkham** [comment]
I'm not going to lie. I don't have lots of time to see through that level of addition. Though I have tried my best to pull in a bunch of missing ufuncs I found in PR ( https://github.com/dask/dask/pull/2315 ) and include them in the tests. If that looks basically fine or has some minor tweaks needed, that should be ok. However, if it feels like it is going to take a lot of review, I may just break out the `sinc` function and focus on that as it is of particular value and time is kind of limited ATM.
